roles/convert2rhel/tasks/activation_keys.yml

```diff
@@ -31,8 +31,7 @@
     organization: "{{ foreman_organization }}"
     name: "{{ foreman_convert2rhel_key_centos7 }}"
     lifecycle_environment: "{{ foreman_convert2rhel_lifecycle_env }}"
-    content_view: "{{ foreman_convert2rhel_content_view }}"
-    subscriptions: "{{ foreman_convert2rhel_org_info['organization']['simple_content_access'] | ternary(omit, foreman_convert2rhel_centos7_subs) }}"
```

Comment:
Are subscriptions unnecessary now? (I'd guess in the non-SCA case, they are still needed?)